config/sync/views.view.stanford_opportunities.yml

```diff
@@ -0,0 +1,3947 @@
+            count: true
+            override: false
+            items_per_page: 25
+          summary:
+            sort_order: asc
+            number_of_records: 0
+            format: default_summary
+          specify_validation: false
+          validate:
+            type: none
+            fail: 'not found'
+          validate_options: {  }
+          break_phrase: true
+          depth: '10'
+          use_taxonomy_term_path: false
+          vocabularies:
+            stanford_event_subject: stanford_event_subject
+        term_node_taxonomy_name_depth_3:
+          id: term_node_taxonomy_name_depth_3
+          table: node_field_data
+          field: term_node_taxonomy_name_depth
+          relationship: none
+          group_type: group
+          admin_label: 'Taxonomy Name: Keywords & Tags'
+          entity_type: node
+          plugin_id: taxonomy_index_name_depth
+          default_action: ignore
+          exception:
+            value: all
+            title_enable: false
+            title: All
+          title_enable: false
+          title: ''
+          default_argument_type: fixed
+          default_argument_options:
+            argument: ''
+          summary_options:
+            base_path: ''
+            count: true
+            override: false
+            items_per_page: 25
+          summary:
+            sort_order: asc
+            number_of_records: 0
+            format: default_summary
+          specify_validation: false
+          validate:
+            type: none
+            fail: 'not found'
+          validate_options: {  }
+          break_phrase: true
+          depth: '10'
+          use_taxonomy_term_path: false
+          vocabularies:
+            stanford_event_keywords: stanford_event_keywords
+      style:
+        type: graphql
+        options: null
+      row:
+        type: graphql_entity
+        options: null
+      defaults:
+        style: false
+        row: false
+        fields: false
+        sorts: false
+        arguments: false
+      display_description: 'List - All Items'
+      display_extenders: {  }
+      graphql_query_name: stanfordOpportunities
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - 'url.query_args:sort_by'
+        - 'url.query_args:sort_order'
+        - 'user.node_grants:view'
+        - user.permissions
+      tags:
+        - 'config:field.storage.node.su_event_alt_loc'
+        - 'config:field.storage.node.su_event_date_time'
+        - 'config:field.storage.node.su_event_dek'
+        - 'config:field.storage.node.su_event_location'
+        - 'config:field.storage.node.su_event_source'
+        - 'config:field.storage.node.su_event_subheadline'
+        - 'config:field.storage.node.su_event_type'
+  more_opportunities_block:
+    id: more_opportunities_block
+    display_title: 'Explore More Opportunities Block'
```

Comment:
Should the explore more be random? currently every opportunity will display the same "more" contents because the sorting settings.